### PR TITLE
Found some extra styles that were affected by refactoring

### PR DESCRIPTION
**Pull request by mdbenjam**
Recent refactors have messed up the search styling and the claims folder issues styling. I'm moving these into glamor.

**Test Plan**
- [ ] Verify the claims folder detail dropdown displays data in two rows instead of each item on a new line.
- [ ] Verify document search properly highlights results as you search through them.

### CODE REVIEW

**Comment by nickheiner-usds** [comment]
Trying to refactor CSS like

image

File: client/app/reader/PdfPage.jsx [hunk 1]
```diff
@@ -25,6 +26,15 @@ const PAGE_MARGIN_BOTTOM = 25;
 // Base scale used to calculate dimensions and draw text.
 const PAGE_DIMENSION_SCALE = 1;
 
+const markStyle = css({
+  '& mark': {
+    background: '#fff1d2',
```
Comment (nickheiner-usds) [inline client/app/reader/PdfPage.jsx]: We previously defined these colors with names. I wouldn't mind keeping that. The constants could live in `AppConstants.js` in this repo.

File: client/app/reader/ClaimsFolderDetails.jsx [hunk 2]
```diff
@@ -24,27 +30,30 @@ class ClaimsFolderDetails extends React.PureComponent {
...
+              </div>
+              <div>
+                <b>Regional Office</b><br />
+                <span>{`${appeal.regional_office.key} - ${appeal.regional_office.city}`}</span>
+              </div>
             </div>
-            <div className="usa-width-one-whole claims-folder-issues">
-              <b>Issues</b><br />
-              <IssueList appeal={appeal} />
+            <div {...rowDisplay}>
```
Comment (nickheiner-usds) [inline client/app/reader/ClaimsFolderDetails.jsx]: Why is this necessary?

Comment (mdbenjam) [reply client/app/reader/ClaimsFolderDetails.jsx]: Hmmm that's good point! I'll remove it.

File: client/app/reader/PdfPage.jsx (hunk 1 above)
Comment (mdbenjam) [reply client/app/reader/PdfPage.jsx]: I prefer that as well. I was hoping to add ALL THE COLORS to CFT, but I'll add them here for now.

File: client/app/constants/AppConstants.js [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
+import { COLORS as COMMON_COLORS } from '@department-of-veterans-affairs/caseflow-frontend-toolkit/util/StyleConstants';
 
 // poll dependency outage every 2 minutes
 export const DEPENDENCY_OUTAGE_POLLING_INTERVAL = 120000;
 export const LONGER_THAN_USUAL_TIMEOUT = 10000;
 export const CERTIFICATION_DATA_POLLING_INTERVAL = 5000;
 export const CERTIFICATION_DATA_OVERALL_TIMEOUT = 180000;
 
+export const COLORS = {
+  ...COMMON_COLORS,
```
Comment (nickheiner-usds) [inline client/app/constants/AppConstants.js]: NICE
